templates/wechatgame/first-screen.js

```diff
@@ -385,6 +388,9 @@ function tick() {
 
 function end() {
     return setProgress(1).then(() => {
+        gl.enable(gl.SCISSOR_TEST);
+        gl.enable(gl.CULL_FACE);
+        gl.enable(gl.DEPTH_TEST);
         cancelAnimationFrame(rafHandle);
```

Comment:
Why need to use GL api to draw in firstscreen? What's the difference between first screen and the original splash screen?